src/utilities/theme/tokens.ts

```diff
@@ -49,12 +48,6 @@ const Overrides = {
   badgeFontWeight: '500',
 };
 
-export const Tokens = {
-  ...BorderRadius,
-  ...Shadow,
-  ...Overrides,
-};
```

Comment:
Why is this a problem?

Reply:
`const Tokens = {...BorderRadius, ...Shadow, ...Overrides}` is also considered potentially sideeffecty by rollup - because the spread results in calls to properties, which may be a getter and thus trigger a side effect. Which means that these variables and everything required to build them - lots of our color manipulation utilities don't get tree-shaken.